tasks/caso-config.yaml

```diff
@@ -1,208 +1,220 @@
 ---
+- set_fact:
```

Comment:
This seems like bad style to me.
Why is a fact being set, instead of a variable being pulled in from one of the vars files?

If this fact is OS dependent, then make it an OS-dependent fact, like 
```yaml
caso_conf:
  redhat: path_for_redhat
  debian: path_for_redhat
```

and look them up accordingly. This would remove the need for the conditionals as well, and make the code easier to read.

Reply:
OK, refactoring